root/edit/details/merge_events.tt

```diff
@@ -2,13 +2,13 @@
   <tr>
     <th>[% l('Merge:') %]</th>
     <td>
-      [%~ INCLUDE 'components/events-list.tt' events=edit.display_data.old checkboxes='' no_ratings=1 ~%]
+      [%~ React.embed(c, 'components/EventsList', { events => edit.display_data.old, noRatings => 1}) ~%]
     </td>
   </tr>
   <tr>
     <th>[% l('Into:') %]</th>
     <td>
-      [%~ INCLUDE 'components/events-list.tt' events=edit.display_data.new checkboxes='' no_ratings=1 ~%]
+      [%~ React.embed(c, 'components/EventsList', { events => [edit.display_data.new], noRatings => 1}) ~%]
```

Comment:
Since EventsList expects an array, events here is made into one. If a different solution would be preferred, let me know!

Reply:
I think it's what the TT is doing too, so sure. :) (The TT is probably compiled into something like `my @events = edit->display_data->{new}` which creates a list of one item).